CHANGELOG.md

```diff
@@ -1099,7 +1099,9 @@ For `AuthAccount`:
   - `monaco-languageclient-cadence`: Language Server Protocol client for the the Monaco editor
   - `cadence-parser`: The Cadence parser
 
-  In addition, there are also examples for the [language server](https://github.com/onflow/cadence/tree/master/npm-packages/cadence-language-server-demo) and the [parser](https://github.com/onflow/cadence/tree/master/npm-packages/cadence-parser-demo) that demonstrate the use of the packages.
+  In addition, there is also an example for the [language server](https://github.com/onflow/cadence/tree/master/npm-packages/cadence-language-server-demo) 
+  <!-- and the [parser](https://github.com/onflow/cadence/tree/master/npm-packages/cadence-parser-demo) that demonstrate the use of the packages.
+  >>
```

Comment:
```suggestion
  In addition, there is also [an example for the language server](https://github.com/onflow/cadence/tree/master/npm-packages/cadence-language-server-demo), and an [AST exploration tool](https://github.com/onflow/cadence/tree/master/tools/astexplorer) that demonstrate the use of the parser package.
```

Reply:
there is no parser-demo in the code, that's why it causes a broken-link issue, therefore I commented parser link. AST exploration is also a good choice to use here instead.